libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dashboardDateRelated.spec.ts

```diff
@@ -0,0 +1,104 @@
+// (C) 2022 GoodData Corporation
+
+import * as Navigation from "../../tools/navigation";
+import { EditMode } from "../../tools/editMode";
+import { DashboardHeader } from "../../tools/dashboardHeader";
+import { DashboardMenu } from "../../tools/dashboardMenu";
+import { Widget } from "../../tools/widget";
+import { DateFilter } from "../../tools/dateFilter";
+import { KpiConfiguration } from "../../tools/kpiConfiguration";
+
+Cypress.Cookies.defaults({
+    preserve: ["GDCAuthTT", "GDCAuthSTT", "_csrfToken"],
+});
+
+Cypress.on("uncaught:exception", (error) => {
+    console.error("Uncaught exception cause", error);
+    return false;
+});
+
+Cypress.Cookies.debug(true);
+
+const dashboardHeader = new DashboardHeader();
+const editMode = new EditMode();
+const dashboardMenu = new DashboardMenu();
+
+describe("Dashboard Date Related", { tags: ["pre-merge_isolated_bear"] }, () => {
+    beforeEach(() => {
+        cy.login();
```

Comment:
should put in the `before()` method  of `index.ts` file or another file under support folder. please help check in other tests

Reply:
not all tests need this

Reply:
Ex: DateFilter
will ask the test script owner why it is not used cy.login.
after getting the answer, i will move cy.login to index.ts in a different PR if needed